ahl_health_apps_analysis/pipeline/refine.py

```diff
@@ -0,0 +1,157 @@
+ 'com.decathlon.canaveral.v1',
+ 'app.collx.android',
+ 'com.bleachr.tennisone',
+ 'com.virtualrepetitions.playbook.basketball',
+ 'com.arkeapps.fantasylifecom.tipstersprime.tipster.reviews',
+ 'co.sayshell.halftime',
+ 'com.padelmates',
+ 'com.codexen.fantasyfootball',
+ 'air.com.nbcuni.com.nbcsports.liveextra',
+ 'com.espn.fantasy.lm.football',
+ 'com.foxsports.android',
+ 'com.dazn',
+ 'tv.mycujoo.mycujooplayer',
+ 'weightloss.women.diet.lose_weight - weight loss diet app',
+ 'com.droidinfinity.compareapp',
+ 'uk.co.disciplemedia.cvgnation',
+ 'com.ymca.universalapp',
+ 'com.mindbodyonline.express',
+ 'strong.vibrator.massage.vibration.forwomen',
+ 'com.jgdevlabs.vibra',
+ 'com.mile.mythalassa',
+ 'com.healme',
+ 'nl.deckeron.apps.innerfire']
+
+# drop apps in drop_list 
+for app in drop_list:
+	app_details = app_details[app_details.appId != app]
+
+'''Refining clusters and renaming cluster_names'''
+#re-cluster 0 to 20 and rename cluster 
+app_details[app_details.appId.isin(re_cluster0)] = app_details[app_details.appId.isin(re_cluster0)].replace({'cluster':0},20)
+app_details[app_details.appId.isin(re_cluster0)] = app_details[app_details.appId.isin(re_cluster0)].replace({'cluster_names':'fertility-health'},'Cycle Tracker')
+
+#re-name cluster 0 
+app_details[app_details.cluster ==0] = app_details[app_details.cluster ==0].replace({'cluster_names':'fertility-health'},'Manage Health Information and Medication') 
+
+# Recipees cluster 1,7,9
+app_details =app_details.replace({'cluster_names':['cooking-recipe']},'Recipes and Meal Kits')
+app_details =app_details.replace({'cluster_names':'calorie-food'}, 'Dietary Management')
+app_details =app_details.replace({'cluster_names':'keto-recipes'}, 'Weight Loss/Diet')
+
+# Workout: cluster 2,5,12,14,15
+app_details =app_details.replace({'cluster_names':['training-workouts', 
+								 'walking-training', 
+								 'fitness-workouts', 
+								 'workouts-fitness',
+								 'workout-classes']},'Workout')
+# Mental Health: cluster 3,13,16
+app_details =app_details.replace({'cluster_names':['therapy-anxiety', 
+								 'health-mental', 
+								 'mindfulness-meditations']},'Mental Health')
+
+app_details =app_details.replace({'cluster_names':'players-tennis'}, 'Improve Sports Performance and Sports Network')
+
+app_details =app_details.replace({'cluster_names':'drinking-drink'}, 'Hydration Reminder')
+
+app_details =app_details.replace({'cluster_names':'weight-pressure'}, 'Monitor Blood Pressure, Insulin and BMI')
+
+app_details =app_details.replace({'cluster_names':'rate-health'}, 'Digital Health Management')
+
+app_details =app_details.replace({'cluster_names':'nclex-anatomy'}, 'Health Information Source')
+
+app_details =app_details.replace({'cluster_names':'restaurants-order'}, 'Food Delivery')
+
+app_details =app_details.replace({'cluster_names':'hearing-sounds'}, 'Better Sleep Through Sound')
+
+app_details =app_details.replace({'cluster_names':'hypnosis-quit'}, 'Quit Smoking or Drinking')
```

Comment:
`app_details['cluster_names'] = app_details['cluster_names'].map({"players-tennis": "'Improve Sports Performance and Sports Network'", 'nclex-anatomy':'Health Information Source' })`